tracee/tracee.bpf.c

```diff
@@ -227,22 +228,23 @@ struct mount {
 
 /*=================================== MAPS =====================================*/
 
-BPF_HASH(config_map, u32, u32);                     // Various configurations
-BPF_HASH(chosen_events_map, u32, u32);              // Events chosen by the user
-BPF_HASH(pids_map, u32, u32);                       // Save container pid namespaces
-BPF_HASH(args_map, u64, args_t);                    // Persist args info between function entry and return
-BPF_HASH(ret_map, u64, u64);                        // Persist return value to be used in tail calls
-BPF_HASH(bin_args_map, u64, bin_args_t);            // Persist args for send_bin funtion
-BPF_HASH(sys_32_to_64_map, u32, u32);               // Map 32bit syscalls numbers to 64bit syscalls numbers
-BPF_HASH(params_types_map, u32, u64);               // Encoded parameters types for event
-BPF_HASH(params_names_map, u32, u64);               // Encoded parameters names for event
-BPF_ARRAY(file_filter, path_filter_t, 3);           // Used to filter vfs_write events
-BPF_ARRAY(string_store, path_filter_t, 1);          // Store strings from userspace
-BPF_PERCPU_ARRAY(bufs, buf_t, MAX_BUFFERS);         // Percpu global buffer variables
-BPF_PERCPU_ARRAY(bufs_off, u32, MAX_BUFFERS);       // Holds offsets to bufs respectively
-BPF_PROG_ARRAY(prog_array, MAX_TAIL_CALL);          // Used to store programs for tail calls
-BPF_PROG_ARRAY(sys_enter_tails, MAX_EVENT_ID);      // Used to store programs for tail calls
-BPF_PROG_ARRAY(sys_exit_tails, MAX_EVENT_ID);       // Used to store programs for tail calls
+BPF_HASH(config_map, u32, u32);                         // Various configurations
```

Comment:
Please keep the original indentation of the maps if there is no reason to change it